common/configuration/aws_secrets_test.go

```diff
@@ -0,0 +1,69 @@
+package configuration
+
+import (
+	"context"
+	"testing"
+
+	"github.com/TBD54566975/ftl/internal/log"
+
+	"github.com/alecthomas/assert/v2"
+	. "github.com/alecthomas/types/optional"
+)
+
+func TestAWSSecretsBasics(t *testing.T) {
```

Comment:
Do a `if !t.Short() { t.SkipNow("A bit slow") }` here (assuming this takes upwards of 5-10 seconds?)

Reply:
It is fast once localstack is running: `PASS: TestAWSSecretsBasics (0.02s)`